draft-irtf-cfrg-fiat-shamir.md

```diff
@@ -177,8 +252,9 @@ SHAKE128 is a variable-length hash function based on the Keccak sponge construct
 
     -  a hash state interface
 
-    1. h = shake_128(iv)
-    2. return h
+    1. initial_block = iv + b'\00' * (168-64)
```

Comment:
I'm just curious how the `iv` is chosen and why we pad it. Also, is the `iv`  expected to be 64 bytes?

Reply:
@mmaker could you answer this one? I am just copying this over from the poc since it seemed like they had gotten out of date :)

Reply:
More details [here](https://mmaker.github.io/draft-irtf-cfrg-sigma-protocols/draft-irtf-cfrg-fiat-shamir.html#section-4). It appears to be 32 bytes.

Reply:
Assuming the intent is to pad the initial block up to the rate, this looks correct, specifically for SHAKE128. I would suggest making this more clear:
```suggestion
    1. initial_block = iv + b'\00' * 106  # len(iv) + 106 == SHAKE128 rate
```

Reply:
Thanks for looking into that, fixed.
(Once github permissions are fixed, feel free to resolve this thread if you feel that it's addressed?